src/lua/digest.lua

```diff
@@ -267,4 +313,70 @@ m['aes256cbc'] = {
     end
 }
 
+for _, var in ipairs({'32', '64'}) do
+    local xxHash
+
+    local update_fn_name = 'XXH' .. var .. '_update'
+    local digest_fn_name = 'XXH' .. var .. '_digest'
+    local reset_fn_name = 'XXH' .. var .. '_reset'
+    local copy_fn_name = 'XXH' .. var .. '_copyState'
+    local methods = {
+        default_seed = 0,
+
+        update = function(self, str)
+            if type(str) ~= 'string' then
+                error("Usage xxhash" .. var .. ":update(string)")
+            end
+            return builtin[update_fn_name](self, str, #str) == builtin.XXH_OK
+        end,
+        result = function(self)
+            return builtin[digest_fn_name](self)
+        end,
+        clear = function(self, seed)
+            if seed == nil then
+                seed = self.default_seed
+            end
+            return builtin[reset_fn_name](self, seed) == builtin.XXH_OK
+        end,
```

Comment:
Would be a good idea to add some empty lines between the function definitions to simplify the reading.

Reply:
Yes, fixed.